src/containers/AdminDashboard/AdminController.tsx

```diff
@@ -290,10 +305,8 @@ class AdminController {
     let r;
     try {
       r = await this.superagent.post(`${process.env.BackendUrl}/user`)
-        .set('Authorization', `Bearer ${auth.token}`)
-        .set('Accept', 'application/json')
-        .send({
-          email: addAdminEmail,
+        .set('Authorization', `Bearer ${auth.token}`).set('Accept', 'application/json')
+        .send({ email: addAdminEmail,
```

Comment:
the below catch should have a notification in it for 'Failed to Create the Admin User'